Dockerfile

```diff
@@ -100,7 +102,8 @@ RUN apk add --no-cache \
     tzdata \
     unzip \
     which \
-    font-noto-cjk
+    font-noto-cjk \
+    nodejs
```

Comment:
- What is the version of nodejs used here? Could you fix it and add an updatecli manifest (in `./updatecli/updatecli.d/`)  to track its version please?
- Could you share, in the PR description, the additional size resulting with this change (the image is already quite heavy)?

Reply:
OK, let me try it. I do not know how to do it(the updatecli manifest), give me some time.

Reply:
No problem: it is not very well documented on the contributing section so there is no strong expectations.
My message was to let you know that it is needed  but I may help.
I suggest that you try by checking the existing manifests.

Reply:
I updated the code. But it is not the final work. Give me some time.

Reply:
I updated the code to add a test case and manifest for Node.js. Please check it. If it has some errors, please help me to update it.

I tried to build it many times but it always threw errors.  Please help me to check the additional size resulting with this change. Thanks.

Reply:
WDYT about using a [Multi stage build]() with the official NodeJS image (instead of instaling the nodejs package) in which you install the NPM packages, and you retrieve only the required directories/files (explained in https://medium.com/geekculture/how-to-install-a-specific-node-js-version-in-an-alpine-docker-image-3edc1c2c64be for a "pure" NodeJS installation).

It would allow faster builds (node js / npm run in parallel of the rest), and a fixed controlled version of nodejs.

Reply:
Let me try it.